src/apps/Permissions/Screen.js

```diff
@@ -1,10 +1,30 @@
 import React from 'react'
+import PropTypes from 'prop-types'
 import styled from 'styled-components'
 import { springs } from '@aragon/ui'
 import { Transition, animated } from 'react-spring'
 
 const SCREEN_SHIFT = 0.05
 
+const Main = ({ children, opacity, left, active }) => (
```

Comment:
Not super worried about this component getting prop types, so it's likely nicer just to leave it as part of `Screen`.

Reply:
I know, but if I did not give `Main` its own `propTypes` I could not get the warning to go away:

![image](https://user-images.githubusercontent.com/3072458/48253855-a857d580-e408-11e8-8772-d6df6e253687.png)